jplag/src/main/java/jplag/CLI.java

```diff
@@ -20,7 +21,21 @@
  */
 public class CLI {
 
-    private static final String DESCRIPTION = "JPlag - Detecting Software Plagiarism";
+    private static final String[] DESCRIPTIONS = {
+            "Detecting Software Plagiarism",
+            "Archaeological Playground",
```

Comment:
How about "Software-Archaeological"

Reply:
i thought the playground thing would be less direct :D